### PR TITLE
[DWS] add `resource/opentelekomcloud_dws_cluster_v1.go`

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
## Summary of the Pull Request
New resource opentelekomcloud_dws_cluster_v1

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.
* [x] Release notes added.

## Acceptance Steps Performed

```
=== RUN   TestAccDwsClusterV1_basic
=== PAUSE TestAccDwsClusterV1_basic
=== CONT  TestAccDwsClusterV1_basic
--- PASS: TestAccDwsClusterV1_basic (702.69s)
PASS


Debugger finished with the exit code 0
```

### CODE REVIEW

**Review by Aloento — DISMISSED** [review]

File: opentelekomcloud/acceptance/dws/resource_opentelekomcloud_dws_cluster_v1_test.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+					testAccCheckDwsV1ClusterExists(resourceInstanceName, &cls),
+					resource.TestCheckResourceAttr(resourceInstanceName, "name", clusterName),
+					resource.TestCheckResourceAttr(resourceInstanceName, "number_of_node", "3"),
+				),
+			},
+			// {
+			// 	Config: testAccDwsV1ClusterUpdated(clusterName),
+			// 	Check: resource.ComposeTestCheckFunc(
+			// 		resource.TestCheckResourceAttr(resourceInstanceName, "name", clusterName),
+			// 		resource.TestCheckResourceAttr(resourceInstanceName, "number_of_node", "6"),
```
Comment (vladimirvshivkov) [inline opentelekomcloud/acceptance/dws/resource_opentelekomcloud_dws_cluster_v1_test.go]: not working?

File: opentelekomcloud/services/dws/resource_opentelekomcloud_dws_cluster_v1.go [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+				Computed:     true,
+			},
+			"number_of_cn": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ValidateFunc: validation.IntBetween(2, 20),
+				Default:      3,
+				ForceNew:     true,
+			},
+			"enterprise_project_id": {
```
Comment (artem-lifshits) [inline opentelekomcloud/services/dws/resource_opentelekomcloud_dws_cluster_v1.go]: remove please

**Review by artem-lifshits — CHANGES_REQUESTED** [review]

**Review by Aloento — DISMISSED** [review]

**Review by Aloento — APPROVED** [review]

File: opentelekomcloud/services/dws/resource_opentelekomcloud_dws_cluster_v1.go (hunk 2 above)
Comment (anton-sidelnikov) [reply opentelekomcloud/services/dws/resource_opentelekomcloud_dws_cluster_v1.go]: done

**Review by artem-lifshits — APPROVED** [review]

File: opentelekomcloud/acceptance/dws/resource_opentelekomcloud_dws_cluster_v1_test.go (hunk 1 above)
Comment (anton-sidelnikov) [reply opentelekomcloud/acceptance/dws/resource_opentelekomcloud_dws_cluster_v1_test.go]: uncommented, it is working but not often

**Review by bakhterets — COMMENTED** [review]
viewed

**Pull request merged**